include/pmacc/nvidia/functors/Min.hpp

```diff
@@ -38,13 +38,13 @@ namespace functors
         template<typename Dst, typename Src >
         DINLINE void operator()(Dst & dst, const Src & src) const
         {
-            dst = algorithms::math::max(dst, src);
+            dst = math::max(dst, src);
         }
 
         template<typename Dst, typename Src, typename T_Acc >
         DINLINE void operator()(const T_Acc &, Dst & dst, const Src & src) const
         {
-            dst = algorithms::math::max(dst, src);
+            dst = math::max(dst, src);
```

Comment:
Should be calling `math::min`, not `math::max` ? Was it a bug before?

Reply:
Well spotted, yes looks like it was a bug before.